wp-content/plugins/wporg-learn/js/course-data/src/style.scss

```diff
@@ -0,0 +1,42 @@
+/**
+ * The following styles get applied both on the front of your site
+ * and in the editor.
+ *
+ * Replace them with your own styles or remove the file completely.
+ */
+
+.wp-block-wporg-learn-course-data {
+	border-top: 1px solid #e2e4e7;
+	list-style: none;
+	margin: 0 0 2rem 0;
+
+	> li {
+		display: flex;
+		justify-content: space-between;
+		border-bottom: 1px solid #e2e4e7;
+		padding: 0.5rem 0 !important;
```

Comment:
This `!important` breaks the print styling. I can see it's been added to override the `.lp-details` styling, so I think you can remove that wrapper from sidebar instead.

```suggestion
		padding: 0.5rem 0;
```

![Screen Shot 2022-12-01 at 4 28 52 PM](https://user-images.githubusercontent.com/1017872/204959209-6d6df745-155e-4206-b0d4-50ddb6f70858.jpg)